lib/console.js

```diff
@@ -162,6 +162,13 @@ Console.prototype.dir = function dir(object, options) {
 };
 
 
+Console.prototype.dirxml = function dirxml(...data) {
+  for (const item of data) {
+    Console.prototype.dir.call(this, item);
```

Comment:
As far as I know, `console` functions must not be called without a proper context, so `this.dir(item)` should work.

Reply:
I'll just check this and make sure it works! Thanks!